uint32 height
uint32 width
float64[9] k_image
string mesh_path
string map_serialized_path